cob_default_env_config/package.xml

```diff
@@ -13,7 +13,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roslaunch</build_depend>  
+  <build_depend>roslaunch</build_depend>
```

Comment:
roslaunch is only a build_depend or also an exec_depend? For other packages (like cob_bringup or cob_default_robot_config)  roslaunch is a depend (build+exec) , what is the difference?